src/pages/clouddocs/release-notes.md

```diff
@@ -16,8 +16,16 @@ toc: true
 
 The following sections document the new features and changes that were included for each release of the Annotator for Clinical Data service.
 
+## May 2023
+
+- Fixes a problem introduced in the April release that affected ACD concurrent processing.
+- Fixes an exception introduced in the April release that occurred in the Attribute Detection service processing derived concepts.
+
 ## April 2023
 
+- The naming convention for environment variables has been changed from com_ibm_watson_health_* to clinical_data_annotator_\*. Any deployment that was setting com_ibm_watson_health_* environment variables should switch to the clinical_data_annotator_* naming convention.
```

Comment:
Formatting seems to be messed up due to the asterisk, or maybe that's just the way it is presented in the review? I didn't load your branch in gatsby to check.

Reply:
@cagrev It just shows up that way in the review, it works fine in preview.  "*." signals markdown to italicize so I had to escape it.